.github/workflows/weekly.yml

```diff
@@ -30,23 +30,20 @@ jobs:
     strategy:
       matrix:
         flink_branches: [{
-          flink: 1.18-SNAPSHOT,
-          jdk: '8, 11, 17',
-          branch: main
-        }, {
           flink: 1.19-SNAPSHOT,
           jdk: '8, 11, 17, 21',
           branch: main
         }, {
           flink: 1.20-SNAPSHOT,
           jdk: '8, 11, 17, 21',
           branch: main
-        }, {
-          flink: 1.18.1,
-          branch: v3.0
         }, {
           flink: 1.19.0,
           branch: v3.0
+        }, {
+          flink: 1.20.0,
+          jdk: '8, 11, 17, 21',
```

Comment:
```suggestion
```
since flink now is 19+ we could remove it from here
and put below 
```
jdk_version: ${{ matrix.flink_branches.jdk || '8, 11, 17, 21' }}
```

Reply:
nice catch!